lavalink/websocket.py

```diff
@@ -128,7 +130,7 @@ def connected(self):
         elif event_type == 'TrackExceptionEvent':
             event = TrackExceptionEvent(player, player.current, data['error'])
         elif event_type == 'WebSocketClosedEvent':
-            event = WebSocketClosedEvent(data["guildId"], data["code"], data["reason"], data["byRemote"])
+            event = WebSocketClosedEvent(data['guildId'], data['code'], data['reason'], data['byRemote'])
```

Comment:
As per above comment regarding this event, perhaps player should be the first parameter?